.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
   - env: RUNTYPE=go.lint
   - env: RUNTYPE=release
     if: tag IS present OR (type != pull_request AND branch = master)
+  - env: RUNTYPE=doc
+    if: branch = master
```

Comment:
The doc is versioned since it lives in the same repo as the stack. It follow the same release cycle.
All content relating to features still not released should contain `draft: true` in their `FrontMatter` (the page header) to avoid being published before the release.

Maintaining documentation is a bit different than code. In some cases, you will want to make changes to the current doc version (ex: typos, found a bug, need more detailed explanation..). Which is why we also need to have the CI running for the latest doc changes.

We can think of a `latest/nightly` doc, that contains current doc changes.